src/devices/MAX31856/samples/MAX31856.Sample.cs

```diff
@@ -18,8 +18,8 @@
 using MAX31856 sensor = new(device, ThermocoupleType.K);
 while (true)
 {
-    // Reads temperature if the device is not reading properly
     var tempColdJunction = sensor.GetColdJunctionTemperature();
+    // Attempts to read temperature if an error is thrown the temperature will read as the default value of temperature in the Temperature class
```

Comment:
You should adjust this comment: there won't be any error thrown as the TryGetTemperature pattern prevent this. You'll just get false as a result and a Temperature that is not a correct one